jstz_cli/src/debug_api/mod.rs

```diff
@@ -11,9 +12,11 @@ impl DebugApi {
 impl jstz_core::Api for DebugApi {
     fn init(self, context: &mut boa_engine::Context<'_>) {
         let kv_api = kv::KvApi::namespace(context);
+        let account_api = account::AccountApi {}.init(context);
```

Comment:
Is it not possible to have:
```suggestion
        let account_api = AccountApi.init(context);
```